app/components/ui/Navigation/Navigation.jsx

```diff
@@ -99,6 +99,11 @@ class Navigation extends React.Component {
                       About
                     </Link>
                   </li>
+                  {/* <li>
+                    <Link to="/resource-guides">
+                      Resource Guides
+                    </Link>
+                  </li> */}
```

Comment:
I think I'd prefer that we just remove this altogether until we actually want it. Having this here but commented out doesn't really provide any information to help us when we actually want to add this. Also, there's no equivalent change to the mobile version of the navigation menu, so it might even cause us to forget to add it to mobile later.